docs/_static/vcr/vcr_v2.yaml

```diff
@@ -487,6 +492,18 @@ components:
           description: RFC3339 time string until when the credential is valid.
           type: string
           example: "2012-01-02T12:00:00Z"
+        credentialSubject:
+          $ref: '#/components/schemas/CredentialSubject'
+        withStatusList2021Revocation:
+          description: |
+            Add a credentialStatus with statusPurpose 'revocation' to the issued credential. This allows a credential to 
+            be revoked using the referenced StatusList2021Credential. StatusPurpose 'suspension' is not supported (yet). 
+            See https://www.w3.org/TR/2023/WD-vc-status-list-20230427/
+            
+            Credentials with a short lifespan (expiry) are preferred over adding a credentialStatus.
```

Comment:
I think we can simply only support it for did:web. Then `visibility` and `publishToNetwork` are `did:nuts`-specific options, this one `did:web` specific. Makes it easier (less fields to pass) and less confusing (no need to pass irrelevant options) on the calling side.